aion/web/elm/src/User/View.elm

```diff
@@ -28,13 +31,32 @@ userView model =
             text (toString error)
 
 
-renderUserView : Location -> CurrentUser -> Html Msg
-renderUserView location user =
+renderUserView : Model -> CurrentUser -> Html Msg
```

Comment:
I think its better to pass just `model.location` and `model.user.scores` instead of whole `model`.